crates/rethnet_eth/src/remote/methods/hardhat.rs

```diff
@@ -7,7 +7,7 @@ use crate::{
 };
 
 /// Compiler input and output structures used as parameters to Hardhat RPC methods
-pub mod compiler_io;
+pub mod add_compilation_result;
```

Comment:
Does it need to be public?

Reply:
it needed to be public back when it lived in `rethnet_eth`, but now that it's co-located in `rethnet_rpc_server` it doesn't. i removed the "allow missing docs" directive in https://github.com/NomicFoundation/hardhat/pull/3998/commits/c86e0d6425c249174b9ddfa0630dff480663b03b